web-portal/backend/src/utils/viemClient.ts

```diff
@@ -0,0 +1,7 @@
+import { createPublicClient, http } from 'viem';
+import { optimism } from 'viem/chains';
+
+export const viemClient = createPublicClient({
+  chain: optimism,
```

Comment:
What is the best way to update this if we add new chains?

Reply:
Array of chains